### ISSUE TITLE
[Bug]: HerdDumper - error using dump with vanilla php framework.

### ISSUE THREAD

**Issue by CamKem** [beyondcode/herd-community#784]
### Platform

macOS

### Operating system version

Sonoma 14.5 (23F79)

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.6.1

### PHP Version

8.3.7

### Bug description

When using dump or dd with a framework I have built that is vanilla php, using some of the Laravel conventions, I get the following error - Depreciated: strlen(): Passing null to parameter #1 ($string) of type string is deprecated in phar:///Applications/Herd.app/Contents/Resources/valet/dump.phar/src/HerdDumper.php on line 182. 

The dumps work fine, both in using dd & dump in the browser and in the dump window, however there is this depreciated error. I tried poking around but couldn't find anything obvious.

<img width="789" alt="Screenshot 2024-06-09 at 5 44 18 PM" src="https://github.com/beyondcode/herd-community/assets/112100521/b3149bbd-affc-4226-9740-0b4b79109b40">

### Steps to reproduce

Use Herd to server an app built with a plain php project, use the dd() or dump() functions & in the browser you should see the message.

### Relevant log output

```shell
[09-Jun-2024 17:40:50 Australia/Melbourne] PHP Deprecated:  strlen(): Passing null to parameter #1 ($string) of type string is deprecated in phar:///Applications/Herd.app/Contents/Resources/valet/dump.phar/src/HerdDumper.php on line 182
```

**Comment by mpociot** [comment]
This will be fixed in todays Herd macOS update

**Comment by CamKem** [comment]
> This will be fixed in todays Herd macOS update

Thanks @mpociot - appreciate it!

**Comment by mpociot** [comment]
This is now fixed in Herd 1.7.0
